packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -52,8 +52,6 @@ export const PeoplePaneContent = (props: {
   // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
   const getStrings = () => {
     return localeStrings.strings.call;
```

Comment:
definitely removing the 2nd return is good its not clear why we went from callWithChat strings to call strings?

Reply:
@JamesBurnside from blame this seems to be quote long ago, do you remember?

Reply:
First thing that comes to mind. I think PeoplePane is a component owned by the calling side, so the strings are probably all there. 🤔